server/routes/api/auth.ts

```diff
@@ -107,9 +113,20 @@ router.post("auth.config", async (ctx) => {
 
 router.post("auth.info", auth(), async (ctx) => {
   const { user } = ctx.state;
-  const team = await Team.scope("withDomains").findByPk(user.teamId, {
-    rejectOnEmpty: true,
-  });
+  const sessions = getSessionsInCookie(ctx);
+  const signedInTeamIds = Object.keys(sessions);
```

Comment:
I'm acutely aware this could be considered an IDOR as having the UUID allows you to access the name and avatarUrl, but this feels like a reasonable security tradeoff.

Reply:
you can validate that the email of the current user matches if you want to clean this up?

Reply:
oh you went and did the super fancy thing! I mean you don't have to for the all the sign in teams... like you could rely on the browser data (which might be stale) for everything that doesnt match the current email and calculate the superset

```
all stuff = browser stuff + server stuff
```

Reply:
You could yea, this feels more solid though and accounts for things like the team has been deleted/disabled too